src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -1360,23 +1360,41 @@ def ngens(self):
         """
         return 1
 
-    def random_element(self, degree=(-1,2), *args, **kwds):
+    def random_element(self, degree=(-1, 2), monic=False, _allow_zero=False, *args, **kwds):
         r"""
-        Return a random polynomial of given degree or with given degree bounds.
+        Return a random polynomial of given degree (bounds).
 
         INPUT:
 
         -  ``degree`` - optional integer for fixing the degree
            or a tuple of minimum and maximum degrees. By default set to
            ``(-1,2)``.
```

Comment:
```suggestion
        -  ``degree`` -- (default: ``(-1, 2)``) integer for fixing the degree
           or a tuple of minimum and maximum degrees
```
Let's make the doc into the correct format while modifying it.

Reply:
Is the correct format to not include a full stop at the end of "INPUT:" docstrings?

Reply:
Generally yes. The exception is if you need multiple sentences to describe the input (although there are other ways to do more complicated behaviors).